samples/apps/wayfinder-sample/start.ps1

```
@@ -0,0 +1,111 @@
+
+New-Item -ItemType Directory -Force -Path (Join-Path $ScriptDir "logs") | Out-Null
+
+function Stop-Port {
+    param([int]$Port)
+    try {
+        $conns = Get-NetTCPConnection -LocalPort $Port -ErrorAction Stop
+        $pidsToKill = $conns | Select-Object -Unique -ExpandProperty OwningProcess
+        foreach ($p in $pidsToKill) {
+            if ($p -and $p -ne $PID) { Stop-Process -Id $p -Force -ErrorAction SilentlyContinue }
+        }
+    } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)